js/mmoda.instrument.js

```diff
@@ -302,18 +152,97 @@ function panel_title(srcname, param) {
           current_nb_attempts_after_failed = 0;
         }
         if (query_failed && (current_nb_attempts_after_failed > max_nb_attempts_after_failed)) {
-          mmoda_show_request_failed(data);
+          waitingDialog.hideSpinner();
+          $('#ldialog').find('.progress').hide();
+          reformatted_exit_status_message = data.exit_status.message.replace(/\\n/g, "<br />");
+          reformatted_exit_status_message = reformatted_exit_status_message.replace(/\n/g, "<br />");
+
+          reformatted_error_message = data.exit_status.error_message.replace(/\\n/g, "<br />");
+          reformatted_error_message = reformatted_error_message.replace(/\n/g, "<br />");
+          waitingDialog.append('<table class="error-table"><tr><td>' + get_current_date_time() + '</td><td>' + reformatted_exit_status_message + '</td></tr><tr><td></td><td>'
+            + reformatted_error_message + '</td></tr></table>', 'danger');
+          waitingDialog.setClose();
+          add_dispatcher_response_to_feedback_form(data);
         } else if (data.query_status != 'done') {
-          mmoda_update_request_progress(data);
+          if ($('.notice-progress-container').is(":hidden")) {
+            $('.notice-progress-container').show();
+          }
+          waitingDialog.showLegend();
+          previous_summary = '';
+
+          if (data.products.hasOwnProperty('input_prod_list')) {
+            data_units = data.products.input_prod_list;
+          }
+
+          if (typeof messages !== 'undefined') {
+            previous_summary = messages.summary;
+          }
+          messages = get_server_message(data, data_units);
+          current_summary = messages.summary;
+          messages.summary = get_current_date_time() + messages.summary;
+          if (current_summary != previous_summary) {
+            waitingDialog.replace(messages);
+            $('#ldialog .summary [data-toggle="tooltip"]').tooltip({
+              trigger: 'hover'
+            });
+          }
+
+          current_ajax_call_params.currentFormData = cloneFormData(current_ajax_call_params.initialFormData);
+          current_ajax_call_params.currentFormData.append('query_status', data.query_status);
+          if (!current_ajax_call_params.currentFormData.has('job_id')) {
+            current_ajax_call_params.currentFormData.append('job_id', job_id);
+            current_ajax_call_params.currentFormData.append('session_id', session_id);
+          }
+          requestTimer = setTimeout(AJAX_call, 5000);
```

Comment:
also this whole piece of code is from the an version of the file, is not in the master
same issue is noticed elsewhere

it seems like an old version of the file has been pushed